site/src/components/OrganizationAutocomplete/OrganizationAutocomplete.tsx

```diff
@@ -7,7 +7,13 @@ import { organizations } from "api/queries/organizations";
 import type { AuthorizationCheck, Organization } from "api/typesGenerated";
 import { Avatar } from "components/Avatar/Avatar";
 import { AvatarData } from "components/Avatar/AvatarData";
-import { type ComponentProps, type FC, useState } from "react";
+import {
+	type ComponentProps,
+	type FC,
+	useEffect,
+	useMemo,
```

Comment:
```suggestion
```

not actually being used (need to figure out why biome isn't catching unused imports...)